### PR TITLE
Docs/new evm - add arbitrum instructions

### PR DESCRIPTION
**Pull request by Errorist79**

### CODE REVIEW

**Review by talalashraf — APPROVED** [review]

File: pages/validator/external-chains/arbitrum.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+</CodeBlock>
+},
+{
+title: "Goerli Testnet",
+content: <CodeBlock language="yaml">
+{`[[axelar_bridge_evm]]
+name = "Arbitrum"
+rpc_addr = "http://IP:PORT"
+start-with-bridge = true`}
+</CodeBlock>
```
Comment (milapsheth) [inline pages/validator/external-chains/arbitrum.md:102]:
```suggestion
title: "Mainnet",
content: <CodeBlock language="yaml">
{`[[axelar_bridge_evm]]
name = "arbitrum"
l1_name_chain = "Ethereum"
rpc_addr = "http://IP:PORT"
start-with-bridge = true`}
</CodeBlock>
},
{
title: "Goerli Testnet",
content: <CodeBlock language="yaml">
{`[[axelar_bridge_evm]]
name = "arbitrum"
l1_chain_name = "ethereum-2"
rpc_addr = "http://IP:PORT"
start-with-bridge = true`}
</CodeBlock>
```

Comment (milapsheth) [reply pages/validator/external-chains/arbitrum.md:102]: arbitrum has a specific syntax

File: pages/validator/external-chains/arbitrum.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+}
+]} />
+
+#### Verify sync status
+
+To verify if your node is in sync you can check the latest block from the explorer.
+Compare it with what you have in `docker ps -q | xargs -L 1 docker logs --tail 10 -f`
+
+- [Mainnet Explorer](https://arbiscan.io)
+- [Goerli explorer](https://goerli.arbiscan.io)
```
Comment (milapsheth) [inline pages/validator/external-chains/arbitrum.md:65]:
```suggestion
- [Testnet Explorer](https://goerli.arbiscan.io)
```

**Review by milapsheth — APPROVED** [review]

File: pages/validator/external-chains/arbitrum.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+vald's `config.toml`
+
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="yaml">
+{`[[axelar_bridge_evm]]
+name = "arbitrum"
+l1_name_chain = "Ethereum"
```
Comment (milapsheth) [inline pages/validator/external-chains/arbitrum.md:91]:
```suggestion
l1_chain_name = "Ethereum"
```

**Pull request merged**